velox/exec/tests/PrefixSortTest.cpp

```diff
@@ -147,6 +147,12 @@ TEST_F(PrefixSortTest, singleKey) {
       makeFlatVector<int64_t>({5, 4, 3, 2, 1}),
       makeFlatVector<int32_t>({5, 4, 3, 2, 1}),
       makeFlatVector<int16_t>({5, 4, 3, 2, 1}),
+      makeFlatVector<int128_t>(
+          {5,
+           HugeInt::build(20, 400),
+           HugeInt::build(20, 401),
+           HugeInt::build(20, 402),
+           HugeInt::build(-20, 401)}),
```

Comment:
@skadilover Can we use `HugeInt::parse` instead of HugeInt::build? 
HugeInt::build only supports unsigned integers even though HugeInt should support signed.
I will add a PR to fix these semantics.